server/src/main/java/org/apache/druid/segment/loading/MostAvailableSizeStorageLocationSelectorStrategy.java

```diff
@@ -32,12 +34,18 @@
 public class MostAvailableSizeStorageLocationSelectorStrategy implements StorageLocationSelectorStrategy
 {
   private static final Ordering<StorageLocation> ORDERING = Ordering.from(Comparator
-      .comparingLong(StorageLocation::availableSizeBytes)
-      .reversed());
+                                                                              .comparingLong(StorageLocation::availableSizeBytes)
```

Comment:
Please roll back useless code formatting.

Reply:
It's accidentally formatted by IDEA. Been fixed.